test/common/schemaform/definitions/nonMilitaryJobs.unit.spec.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import ReactTestUtils from 'react-addons-test-utils';
+
+import { DefinitionTester } from '../../../util/schemaform-utils.jsx';
+import { schema, uiSchema } from '../../../../src/js/common/schemaform/definitions/nonMilitaryJobs';
+
+describe('Schemaform definition nonMilitaryJobs', () => {
```

Comment:
The 5490 page has a test for adding another employment history row. Can you move that here?